--- diction-1.09/diction.texi.in.orig	2006-03-11 11:58:03.000000000 +0100
+++ diction-1.09/diction.texi.in	2006-03-11 11:58:31.000000000 +0100
@@ -88,6 +88,7 @@
 * Introduction::                Introduction.
 * Invoking::                    Invoking @command{diction}; description of options.
 * Diagnostics::                 Exit status returned by @command{diction}.
+* Author::                      Who wrote diction?
 * Reporting Bugs::              Reporting Bugs.
 * Concept Index::               A menu with all the topics in this manual.
 * Index::                       A menu with all @command{diction} commands
@@ -245,8 +246,8 @@
 of the bug behavior, and the input to `diction' that triggered the bug.
 @c }}}
 @c {{{ Author
-@c node Author
-@c chapter Author
+@node Author
+@chapter Author
 
 This program is GNU software, copyright 1997-2005 Michael Haardt
 @email{michael@@moria.de}.
