diff -urN diction-1.03.orig/Makefile.in diction-1.03/Makefile.in
--- diction-1.03.orig/Makefile.in	2003-04-08 23:28:58.000000000 +0200
+++ diction-1.03/Makefile.in	2003-08-09 20:20:38.000000000 +0200
@@ -35,21 +35,21 @@
 		getopt.o getopt1.o $(LIBM) $(LIBS)
 
 install:	all
-		[ -d @bindir@ ] || @INSTALL@ -m 755 -d @bindir@
-		@INSTALL@ diction @bindir@/diction
-		@INSTALL@ style @bindir@/style
-		@INSTALL@ -m 755 -d @datadir@/diction
-		@INSTALL@ -m 644 de @datadir@/diction/de
-		@INSTALL@ -m 644 en @datadir@/diction/en
-		(cd @datadir@/diction; rm -f C; ln en C)
-		[ -d @mandir@/man1 ] || @INSTALL@ -m 755 -d @mandir@/man1
-		@INSTALL@ -m 644 diction.1 @mandir@/man1/diction.1
-		@INSTALL@ -m 644 style.1 @mandir@/man1/style.1
+		[ -d $(DESTDIR)@bindir@ ] || @INSTALL@ -m 755 -d $(DESTDIR)@bindir@
+		@INSTALL@ diction $(DESTDIR)@bindir@/diction
+		@INSTALL@ style $(DESTDIR)@bindir@/style
+		@INSTALL@ -m 755 -d $(DESTDIR)@datadir@/diction
+		@INSTALL@ -m 644 de $(DESTDIR)@datadir@/diction/de
+		@INSTALL@ -m 644 en $(DESTDIR)@datadir@/diction/en
+		(cd $(DESTDIR)@datadir@/diction; rm -f C; ln en C)
+		[ -d $(DESTDIR)@mandir@/man1 ] || @INSTALL@ -m 755 -d $(DESTDIR)@mandir@/man1
+		@INSTALL@ -m 644 diction.1 $(DESTDIR)@mandir@/man1/diction.1
+		@INSTALL@ -m 644 style.1 $(DESTDIR)@mandir@/man1/style.1
 		make install-po-@USE_NLS@
 install-po-no:
 install-po-yes:	$(CATALOGS)
 		for cat in $(CATALOGS); do \
-		  dir=$(localedir)/`basename $$cat .mo`/LC_MESSAGES; \
+		  dir=$(DESTDIR)$(localedir)/`basename $$cat .mo`/LC_MESSAGES; \
 		  [ -d $$dir ] || @INSTALL@ -m 755 -d $$dir; \
 		  @INSTALL@ -m 644 $$cat $$dir/diction.mo; \
 		done
